test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -114,23 +114,14 @@ var _ = table.DescribeTable("Test latency measurement tools tests", func(testGro
 			if err != nil {
 				testlog.Error(err.Error())
 			}
-			Expect(string(output)).NotTo(MatchRegexp(unexpectedError), "Unexpected error was detected in a positve test")
+			Expect(string(output)).NotTo(MatchRegexp(unexpectedError), "Unexpected error was detected in a positive test")
 			//Check runtime argument in the pod's log only if the tool is expected to be executed
 			ok, matchErr := regexp.MatchString(success, string(output))
 			if matchErr != nil {
 				testlog.Error(matchErr.Error())
 			}
 			if ok {
-				var commandRegex string
-				if test.toolToTest == oslat {
-					commandRegex = fmt.Sprintf("Running the oslat command with arguments .*--duration %s", test.testRuntime)
-				}
-				if test.toolToTest == cyclictest {
-					commandRegex = fmt.Sprintf("running the cyclictest command with arguments .*-D %s", test.testRuntime)
-				}
-				if test.toolToTest == hwlatdetect {
-					commandRegex = fmt.Sprintf("running the hwlatdetect command with arguments .*--duration %s", test.testRuntime)
-				}
+				commandRegex := fmt.Sprintf("%s command with arguments .*--duration %s", test.toolToTest, test.testRuntime)
```

Comment:
I don't think we need this change in 4.11

Reply:
IIUC, NTO still uses the tool runners from the cnf-features deploy, and the master branch of the latter is pointing to 4.11, thus the original modification of this arg was actually post 4.10 GA on the master branch and was backported to 4.10 branch. check the actual runner args here for 4.11 (from master):
https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/pod-utils/cyclictest-runner/main.go#L52
original PR: https://github.com/openshift-kni/cnf-features-deploy/pull/971

Reply:
cc @Tal-or

Reply:
>NTO still uses the tool runners from the cnf-features deploy,

NTO is not using the runner. 
It's cnf-features deploy which is using the runner to run the tests. The tests are consumed from PAO (which is now part of NTO)

Yes. we have the ```--duration``` presented in 4.11 as well

Reply:
so there is still dependency in cnf-features-deploy to NTO right? otherwise how do we know what args are passed to each tool command